test/TestAssets/TraceCollectorLoader/UnitTest1.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Linq;
+
+namespace TraceCollectorLoader
```

Comment:
The name of the project is slightly misleading. Rename it to what is actually doing, something related to GetAssemblies.

Reply:
Updated